app/views/users/new.html.erb

```diff
@@ -17,8 +17,8 @@
   <%= label_tag :email %>
   <%= text_field_tag :email, "#{@user.email}" %>
 
-  <%= label_tag :password_digest %>
-  <%= password_field_tag :password_digest, "#{@user.password_digest}" %>
+  <%= label_tag :password %>
+  <%= password_field_tag :password, "#{@user.password_digest}" %>
 
```

Comment:
Just to confirm - the reference to @user.password_digest can stay as is b/c it's pulling out of the DB field, right? Should that even be there? Are we supposed to display the password back to the user at all?